contracts/modules/TransactionManager.sol

```diff
@@ -35,23 +36,37 @@ abstract contract TransactionManager is BaseModule {
     bytes4 private constant ERC1155_INTERFACE = ERC1155_RECEIVED ^ ERC1155_BATCH_RECEIVED;
 
     struct Call {
-        address to;
-        uint256 value;
-        bytes data;
-        bool isSpenderInData;
+        address to;         // the target of the call
+        uint256 value;      // the ETH to transfer
+        bytes data;         // the data payload
+        bool isTokenCall;   // true if the target is a token
```

Comment:
I'd rename `isTokenCall` to `isStandardTokenCall` and update the comment to "true if target is a token and data is a standard ERC20/721/1155 method" since SNX, aTokens, etc have non-ERC20 methods (e.g. `redeem(uint256)`) that would be called with `isTokenCall = false`

Reply:
I've updated the comment but not the parameter name to keep it short. I believe the updated comment is sufficient to make its function clear.